Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -125,6 +127,9 @@ type BotInProject = {
   projectId: string;
   name: string;
   isRemote: boolean;
+  isRootBot: boolean;
+  diagnostics: Diagnostic[];
+  [key: string]: any;
```

Comment:
Please don't add arbitrary fields to things. This ruins type-checking. Just add the fields you plan on using. If you need some kind of arbitrary key-value store on a bot, then give it a good name and make its type `{[key: string]: any}`.